### DISCUSSION TITLE
ImportError: cannot import name 'AttentionalAggregation' from 'torch_geometric.nn' when running examples.

### DISCUSSION THREAD

**Discussion by Accelerator-thu** [pyg-team/pytorch_geometric#5493] [Q&A] [answered]
Hi there,
    My installation is listed as follows:

pytorch                   1.8.0               py3.8_cpu_0  [cpuonly]  pytorch
torch-geometric           2.0.0                    pypi_0    pypi
torch-scatter             2.0.8                    pypi_0    pypi
torch-sparse              0.6.12                   pypi_0    pypi
torchaudio                0.10.0+rocm4.1           pypi_0    pypi
torchvision               0.11.0+cu113             pypi_0    pypi

    After running `python main.py` under the directory ./benchmark/kernel, I got the following error:

Traceback (most recent call last):
  File "main.py", line 10, in <module>
    from global_attention import GlobalAttentionNet
  File "/home/***/pytorch_geometric-master/benchmark/kernel/global_attention.py", line 5, in <module>
    from torch_geometric.nn import AttentionalAggregation, SAGEConv
ImportError: cannot import name 'AttentionalAggregation' from 'torch_geometric.nn' (/home/***/miniconda3/envs/torchpyg/lib/python3.8/site-packages/torch_geometric/nn/__init__.py)

    How should I solve this?

**Comment by EdisonLeeeee** [comment, marked as answer]
`nn.aggr` is supported for PyG>=2.1.0. It seems you need to upgrade your PyG.

**Reply by Accelerator-thu** [reply]
Thanks a lot for that! It worked.
